addon/components/frost-select.js

```diff
@@ -384,11 +384,17 @@ export default Component.extend(PropTypeMixin, {
   // ==========================================================================
   // Events
   // ==========================================================================
-
+  focusIn (event) {
+    this.get('reduxStore').dispatch(openDropDown)
+    // If an onFocus event handler is defined, call it
+    if (this.attrs.onFocus) {
+      this.attrs.onFocus(event)
```

Comment:
Didn't we deprecate usage of `onFocus` on the other core components? It seems we ought to do the same here.